bears/cmake/CMakeLintBear.py

```diff
@@ -21,12 +22,16 @@ class CMakeLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
-                         cmakelint_config: path = '',
-                         ):
+                         cmakefilter_list: typed_list(str) = [],
+                         cmakelint_config: path = ''):
```

Comment:
why are you changing this line??

Reply:
travis logs showed error only in these lines.